$MirOS: ports/net/centericq/patches/patch-firetalk-0_1_firetalk_c,v 1.2 2005/07/05 16:04:57 tg Exp $
--- firetalk-0.1/firetalk.c.orig	Mon Aug 29 08:56:45 2005
+++ firetalk-0.1/firetalk.c	Sun Oct  9 20:10:15 2005
@@ -2431,7 +2431,7 @@ enum firetalk_error firetalk_file_offer(
 	while(p = strstr(cp, "/"))
 	    cp = p+1;
 
-	sprintf(args, "SEND %s %lu %lu %ld", cp, conn->localip, conn->file_head->port, conn->file_head->size);
+	sprintf(args, "SEND %s %lu %lu %ld", cp, conn->localip, (unsigned long)conn->file_head->port, conn->file_head->size);
 	return firetalk_subcode_send_request(conn, nickname, "DCC", args);
 }
 
